config/locales/server.en.yml

```diff
@@ -5,6 +5,7 @@ en:
     kolide_webhook_secret: 'The string to verify incoming Kolide webhooks'
     kolide_verbose_log: 'Enable verbose logging for Kolide plugin'
     kolide_onboarding_topic_id: 'Topic with Kolide onboarding instructions for users'
+    kolide_admin_group_name: 'The Kolide admins group name where user device problems will be reported'
```

Comment:
I am not sure why this needs to be added in this PR, was it just missing from a previous one which added the site setting?

Reply:
yes, I didn't add this locale in the original commit.